tripy/tests/spec_verification/op_doc_guide.md

```diff
@@ -0,0 +1,29 @@
+# Introduction
+
+Spec verification is designed to ensure that the datatypes documented in the operator documentation are accurate.
+
+# How to Verify an Operation
+
+To run the verification program on an operation, add the decorator @dtype_info.dtype_info to the operation. The inputs to the decorator will help the verifier determine the constraints on the inputs and the datatypes to verify.
+
+There are four optional inputs for the decorator:
```

Comment:
Shouldn't these be documented in the docstring? If it's in a separate file it's likely to go stale. IMO this file should be more like a high level explanation of why/what we're doing, not a granular explanation of what each parameter means.

Reply:
I was thinking of having one place where you can learn about how the verifier works without having to search for more information, but I can just link to the function so that users can read more about it there.